repository/main/urls.py

```diff
@@ -17,12 +17,36 @@
 from django.conf.urls.static import static
 from django.contrib import admin
 from django.urls import include, path, re_path
+from rest_framework import permissions
+from drf_yasg.views import get_schema_view
+from drf_yasg import openapi
 
+schema = get_schema_view(  # pylint: disable=invalid-name
+    openapi.Info(
+        title="Program repository API",
+        default_version="v1",
+        description="List of available API endpoint for Program repository.",
+    ),
+    public=True,
```

Comment:
I would set this value to `False` to show only those end-points accesible by the user.

I could make log-in works adding in `settings.py` the next lines of code:
```
SWAGGER_SETTINGS = {
    "LOGIN_URL": "/api-auth/login/",
    "LOGOUT_URL": "/api-auth/logout/"
}
```

Reply:
added in https://github.com/Qiskit-Extensions/quantum-serverless/pull/225/commits/d69349292ff90d6bd79c8ccd98a83954c009c818

Reply:
I miss the change about `public = False` to hide end-points not accesible if you are not logged, what do you think, @IceKhan13 ?

Reply:
Good suggestion! Fixed!